src/create-model.jl

```diff
@@ -914,7 +942,22 @@ function create_model(
             sum(
                 graph[a].investment_cost[y] * graph[a].capacity[y] * assets_investment[y, a] for
                 y in Y for a in Ai[y]
-            ) + sum(
+            )
+        )
+
+        assets_fixed_cost = @expression(
+            model,
+            sum(
+                graph[a].fixed_cost[y] *
+                graph[a].capacity[y] *
+                accumulate_capacity_simple_method[y, a] for y in Y for
+                a in investable_assets_using_simple_method[y]
```

Comment:
- Same here, we also want it to work with only initial/existing units that we don't invest in anymore.

Reply:
Latest changes account for the reviewed formulation